package-lock.json

```diff
@@ -2680,10 +2680,10 @@
       "dev": true
     },
     "bulk-data-utilities": {
-      "version": "git+https://github.com/projecttacoma/bulk-data-utilities.git#fb04a13d49357b74e6e8fed55e97d44a4b490c2d",
+      "version": "git+https://github.com/projecttacoma/bulk-data-utilities.git#32be10a86f6fc7386b26ae93df4a16d2944936d4",
```

Comment:
it looks like the commit hash of the bulk-data-utilities branch has been downgraded by a few commits, can you undo this and re-npm install just to make sure that it doesn't actually generate any diffs? Assuming this was accidental

Reply:
good catch, yes this was accidental, not sure what happened